lib/reel/session/store.rb

```diff
@@ -5,19 +5,19 @@ module Reel
   module Session
     class Store
       include Crypto
+      include Celluloid
```

Comment:
see the above comment, since `initialize_session` calls `Store.new`, including celluloid here causes those new Store instances to be actors.